cgi-bin/ljlib.pl

```diff
@@ -54,12 +54,24 @@ BEGIN
     }
 }
 
-# Now set up logging support for everybody else to access
+# Now set up logging support for everybody else to access; this is done
+# very early. We may be called by a test though, which will set the flag,
+# and in that case we disable all the logging.
 use Log::Log4perl;
-
 BEGIN {
-    Log::Log4perl::init_and_watch($LJ::HOME . '/etc/log4perl.conf', 10);
-    Log::Log4perl->get_logger('LJ')->info('ljlib.pl starting up...');
+    if ( $LJ::_T_CONFIG ) {
+        # Tests, don't log
+        my $conf = q{
+log4perl.rootLogger=FATAL, DevNull
+
+log4perl.appender.DevNull=Log::Log4perl::Appender::File
+log4perl.appender.DevNull.filename=/dev/null
+log4perl.appender.DevNull.layout=Log::Log4perl::Layout::SimpleLayout
+        };
+        Log::Log4perl::init( \$conf );
+    } else {
+        Log::Log4perl::init_and_watch($LJ::HOME . '/etc/log4perl.conf', 10);
```

Comment:
Did you address my comment about changing the default log level in log4perl.conf from DEBUG to INFO?  Or did you decide it wasn't necessary?